sound/soc/amd/acp/amd-acp63-acpi-match.c

```diff
@@ -9,21 +9,21 @@
 #include <sound/soc-acpi.h>
 #include "../mach-config.h"
 
-static const struct snd_soc_acpi_endpoint single_endpoint = {
+static struct snd_soc_acpi_endpoint single_endpoint = {
```

Comment:
Do we actually need to remove the const on all of these hard coded entries? At least from the description where we are only creating a match if one doesn't exist that doesn't imply we would be changing an existing hard coded entry?

Reply:
The problem is that I don't know how to convert a const struct to a non-const struct. For example, I will get below compile error if I don't remove the const of cs42l43_endpoints and try to force cast the type when set the .endpoints.
```
sound/soc/intel/common/soc-acpi-intel-arl-match.c:200:38: error: conversion to non-scalar type requested
  200 |                 .endpoints = (struct snd_soc_acpi_endpoint)cs42l43_endpoints,
      |                                      ^~~~~~~~~~~~~~~~~~~~~
```
It would be great if you could provide me some advice. I am more than happy if we don't need to remove the const on all of those hard coded entries.